src/main/java/org/fundacionjala/at15/katas/pokerhands/jimy/Card.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.at15.katas.pokerhands.jimy;
+
+public class Card {
+    private int value;
+    private char suit;
+    private final int ten = 10;
+    private final int elev = 11;
+    private final int twel = 12;
+    private final int thert = 13;
+    private final int ace = 14;
+
+    public Card(String input) {
+        suit = input.charAt(input.length() - 1);
+        String aux = input.substring(0, input.length() - 1);
```

Comment:
The idea of this Card is good, I would move the logic of this constructor to a private class and just call it inside the constructor, so the constructor is simplified